saleor/graphql/order/tests/queries/test_order_with_filter.py

```diff
@@ -1360,3 +1360,66 @@ def test_order_query_with_filter_by_empty_list(
     # then
     content = get_graphql_content(response)
     assert content["data"]["orders"]["totalCount"] == len(orders_from_checkout)
+
+
+def test_order_query_with_filter_checkout_tokens(
+    orders_query_with_filter,
+    staff_api_client,
+    permission_group_manage_orders,
+    order,
+    orders_from_checkout,
+    channel_USD,
+):
+    assert not order.checkout_token
+    assert all([order.checkout_token for order in orders_from_checkout])
+    # given
+    variables = {
+        "filter": {
+            "checkoutTokens": [order.checkout_token for order in orders_from_checkout],
```

Comment:
Do you think this test will make sure your filter is working properly?
You are providing here the `checkout_token` of all existing orders.

Reply:
there is an additional one `order` which is added as a fixture and there are 2 asserts that checks if order.checkout_token is not assigned